### DISCUSSION TITLE
Publishing Private Websites

### DISCUSSION THREAD

**Discussion by samarzayan** [quarto-dev/quarto-cli#3182] [Q&A]
Is there a way to publish my website privately which can not be seen by anyone or comes up in Google searches? Or perhaps add a feature of a password to enter it?

I wish to create a personal knowledge management space in the form of a website to be accessed only by me. 

Thanks in advance!

**Comment by mcanouil** [comment]
Quarto provides few publishing targets for convenience.
As long as you have access, you can render whatever you want to publish and simply upload it to wherever you want.
Quarto cannot possibly handle all possible services, you'll have to refer to your provider documentation.

https://quarto.org/docs/publishing/other.html

Password can be added using htaccess file for example (although this is pointless, if you use public GitHub Pages, GitLab pages, etc).
